index.qmd

```diff
@@ -1,120 +1,410 @@
+  xlab = "Features",
+  ylab = "Features",
+  main = "Correlation Heatmap",
+  colors = colorRampPalette(c("red", "white", "brown"))(100),
+  limits = c(-1, 1),
+  branches_lwd = 0.1,
+  titleX = FALSE,
+  titleY = FALSE,
+        label_names = c("Variable", "Factor", "Value"),
+        fontsize_row = 10, fontsize_col = 10,
+        labCol = colnames(correlation_matrix),
+        labRow = rownames(correlation_matrix),
+        heatmap_layers = theme(axis.line = element_blank())
+)
+
+```
 
-# Results
+ The correlation shows moderately positive correlations between the Age and Pregnancy, and the Insulin and Pregnancy. This indicates that as the age of the patients increased so did the number of pregnancies, also as the number of pregnancies, the quantity of insulin administered to the patients increased likewise .
 
-\[\~200 words\]
+Weak or no correlations can also be observed in the following attributes of the dataset; DiabetesPedigree and Skin Thickness.
 
-Tables and figures (maps and other graphics) are carefully planned to convey the results of your analysis. Intense exploration and evidence of many trials and failures. The author looked at the data in many different ways before coming to the final presentation of the data.
+##### Comparing Outcomes and Variables
 
-Show tables, plots, etc. and describe them.
+Age vs Outcome
 
-```{r, fig.width=6, fig.height=3, fig.cap="Map of completely random data"}
-m <- leaflet(data) %>% 
-  addTiles() %>% 
-  addCircleMarkers(~x, ~y, radius = ~size,color = ~as.factor(category)) %>% 
-  addPopups(~x[2], ~y[2], "Random popup")
-m  # a map with the default OSM tile layer
+```{r fig.width=8, fig.height=4, fig.cap="Graph Showing Correlation of Outcome vs Age", warning=FALSE, message=FALSE}
+ggplot(data = diabetes_data, aes(x = Age)) + geom_histogram(color = "blue", fill = "lightblue") + facet_wrap(~Diabetic) + theme_dark() + ylab("Number of Patients") + labs(title = "Age(s) of Patients")
 ```
 
+0 = Non-diabetic
+
+1= Diabetic
+
+The ages of the patients are skewed to the right with most of the patients being between the ages of 20 to 40.
+
+BMI vs Outcome
+
+```{r fig.width=8, fig.height=4, fig.cap="Plot Showing Correlation Between Outcome vs BMI", warning=FALSE, message=FALSE}
+ggplot(data = diabetes_data, aes(x = BMI)) + geom_histogram(color = "blue", fill = "lightblue") + facet_wrap(~Diabetic) + theme_dark() + ylab("Number of Patients") + labs(title = "BMI of Patients")
+```
+
+Blood Pressure vs Outcome
+
+```{r fig.width=8, fig.height=4, fig.cap="Violin plot Showing Correlation Between Outcome vs BloodPressure"}
+ggplot(diabetes_data, aes(x = factor(Diabetic), y = DiastolicBloodPressure, fill = factor(Diabetic))) +
+  geom_violin() +
+  labs(title = "Patients' Blood Pressure", x = "Diabetes Status", fill = "Diabetes Status" ) +
+  scale_fill_discrete(labels = c("Non-Diabetic", "Diabetic")) +
+  theme_dark()
+```
+
+Visualizing the distribution of blood pressure for each outcome .
```

Comment:
Would be useful to see the boxplot within the Violin plot. I beleive it will had more insights into your results. Great job overall!